packages/amplify-category-auth/src/commands/auth/update.ts

```diff
@@ -53,8 +53,8 @@ export const run = async (context: $TSContext) => {
   }
   const resourceName = await getAuthResourceName(context);
   await checkAuthResourceMigration(context, resourceName);
-  const cliState = new AuthInputState(resourceName);
-  context.updatingAuth = await cliState.loadResourceParameters(context, cliState.getCLIInputPayload());
+  const providerPlugin = context.amplify.getPluginInstance(context, servicesMetadata.Cognito.provider);
```

Comment:
we also need the JSON.parse(providerPlugin.loadResourceParameters(context, 'auth', resourceName))

Reply:
No, I ran it with this and it works. The line that's already there is
```ts
const previouslySaved =
    typeof context.updatingAuth.triggers === 'string' ? JSON.parse(context.updatingAuth.triggers) : context.updatingAuth.triggers;
```
And with this change `context.updateAuth.triggers` will be a string and be properly parsed

Reply:
okay

Reply:
oh wait, I may not have rebuilt in between. Let me check again...

Reply:
yep good catch, I hadn't rebuilt and it is needed. Updating the PR now